impl/src/main/java/org/ehcache/impl/internal/classes/commonslang/ArrayUtils.java

```diff
@@ -44,7 +44,7 @@
     /**
      * An empty immutable {@code Class} array.
      */
-    public static final Class<?>[] EMPTY_CLASS_ARRAY = new Class[0];
+    public static final Class<?>[] EMPTY_CLASS_ARRAY = new Class<?>[0];
```

Comment:
Not fond of modifying these imported classes.

Reply:
Yes... But they still have a compilation warning... Not much I can do